Services/UICore/classes/class.ilTemplate.php

```diff
@@ -1520,6 +1520,15 @@ function getTemplatePath($a_tplname, $a_in_module = false, $a_plugin = false)
 			{
 				$fname = "./".$module_path."templates/default/".basename($a_tplname);
 			}
+		}else if(strpos($a_tplname,"src/UI")===0){
+			if (ilStyleDefinition::getCurrentSkin() != "default")
+			{
+				$fname = "./Customizing/global/skin/".ilStyleDefinition::getCurrentSkin()."/".str_replace("src/UI/templates/default","UI",$a_tplname);
+			}
+			if($fname == "" || !file_exists($fname))
+			{
+				$fname = $a_tplname;
+			}
```

Comment:
I guess we should discuss this on JF, right? I'm not 100% sure i understand the magic.